subprojects/docs/src/docs/release/notes.md

```diff
@@ -146,6 +146,11 @@ The following are the newly deprecated items in this Gradle release. If you have
 
 ## Potential breaking changes
 
+### maven-publish and ivy-publish mirror multi-project behavior
+
+When using the `java` plugin, all `compile` and `runtime` dependencies will now be mapped to the `compile` scope, i.e. "leaked" into the consumer's compile classpath. This is in line with how
```

Comment:
It only accidentally worked and only for the java-library plugin, in which case people wouldn't use compile and runtime anyway. For the java plugin we're changing those dependencies from all runtime to all compile, which at least means that compilation against such a component will work now.